src/dowser/__init__.py

```diff
@@ -493,3 +494,21 @@ def setup(app: aiohttp.web.Application, **kwargs):
     bind_path = kwargs.get('bind_path') or '/dowser/'
     app['dowser'] = {'bind_path': bind_path}
     app.add_subapp(bind_path, dowser_blueprint)
+
+
+def spawn(host="127.0.0.1", port=None):
+    if not port:
+        port = 9000 + os.getpid() % 1000
```

Comment:
How it is supposed to be used? Exact port is not determined here, so user wouldn't know which port was chosen by server: the function gives no feedback neither as output on console nor as return value. I'd propose to let the function just use something like 8080 as a default.

Reply:
Fixed, set the default port 8080 for the method.

> How it is supposed to be used? 

The main idea was - In a multiprocessor application you would not have to write the logic of choosing a port, but simply add a call to the spawn method and different processes would listen on different ports.
Since the logic of choosing the port fits in one line, I decided that it was really more correct to take it out of the method.